example/kustomize-krm/cert-manager/generator.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: khelm.mgoltzsche.github.com/v2
+kind: ChartRenderer
+metadata:
+  name: cert-manager
+  namespace: cert-manager
+  annotations:
+    config.kubernetes.io/function: |
+        container:
+          image: mgoltzsche/khelm:latest
+          network: true
+    config.kubernetes.io/local-config: "true"
```

Comment:
I think this annotation can be removed since it was only useful when declaring a KRM function for kpt which otherwise treated the `generator.yaml` as a regular resource, adding it to the generated static manifest as I remember. When using kustomize, I don't think it has any effect.